src/deepsparse/benchmark_model/benchmark_model.py

```diff
@@ -145,7 +145,7 @@ def parse_args():
         "--scenario",
         type=str,
         default="async",
-        choices=["async", "sync"],
+        choices=["async", "sync", "elastic"],
         help=(
             "Choose between using a sync/async scenarios. This is similar to the "
             "single-stream/multi-stream scenarios. Default value is async."
```

Comment:
@SageMoore could we update docs here to reflect the added elastic and a small blurb on what it does differently?

Reply:
Definitely

Reply:
LGTM